core/deployment/src/main/java/io/quarkus/deployment/builditem/ArchiveRootBuildItem.java

```diff
@@ -98,17 +89,6 @@ private ArchiveRootBuildItem(Builder builder, QuarkusBuildCloseablesBuildItem bu
         this.archiveRoot = this.rootDirs.iterator().next();
     }
 
-    /**
-     * If this archive is a jar file it will return the path to the jar file on the file system,
-     * otherwise it will return the directory that this corresponds to.
-     *
-     * @deprecated in favor of {@link #getPaths()}
-     */
-    @Deprecated
-    public Path getArchiveLocation() {
-        return archiveLocation;
-    }
-
```

Comment:
Here is a variant of the method that will keep the platform TS passing, given that the test paths are added first we are returning the last on the list
````
    /**
     * If this archive is a jar file it will return the path to the jar file on the file system,
     * otherwise it will return the directory that this corresponds to.
     *
     * @deprecated in favor of {@link #getPaths()}
     */
    @Deprecated
    public Path getArchiveLocation() {
        final Iterator<Path> i = paths.iterator();
        Path last = i.next();
        while (i.hasNext()) {
            last = i.next();
        }
        return last;
    }
````

Reply:
The good thing is removing `public Path getArchiveRoot()` does not actually break the platform.